packages/raystack/components/link/link.module.css

```
@@ -1,9 +1,39 @@
+  }
+
+  /* Press dims at the snappy press tier; releasing eases back at the base
+     (fast) tier above — quick press, gentle release. */
+  .link:active {
+    transition: opacity var(--rs-duration-press) var(--rs-ease-out);
+  }
 }
 
 .link:hover {
-  opacity: 0.8;
+  text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)